README.md

```diff
@@ -32,6 +32,7 @@ which pip #or which python
 # The output should look like this:
 # /.conda/envs/yourenvname/bin/pip
 ```
+**Note**: In case the environment is not properly activated, i.e., `which pip` outputs a path not as mentioned above, try `conda init`. Close and reopen a new terminal and do `conda activate <yourenvname>`.
 
 3. You will need to install the GromacsWrapper module. You can
 download the [source code](https://github.com/Becksteinlab/GromacsWrapper/releases) for version 0.8.0 (latest) and install using the following command:
```

Comment:
The current version requires that we use this custom gromacswrapper: 
`pip install --no-cache-dir git+https://github.com/whitead/GromacsWrapper`

Reply:
This branch was created from older version. New version has already updated the installation tutorial. I will make sure to keep the correct version when merging.